### PR TITLE
Create researchfish-updates-2021.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
This is a PR the mega researchfish update (some sensitive things removed e.g. testimonials). It has been checked over by @lottycoupat and @angelsaur and should be ready for your review @KirstieJane! 

* Where I have made a correction to the HackMD I have also made a correction to the Researchfish portal
* I've taken out some elements of the report which didn't seem right to publish on GitHub, such as personal testimonials - would be great if you could look for anything else that wouldn't make sense or be right to publish here
* Any other formatting, or typos, or grammar to correct? 

Thanks!

### CODE REVIEW

**Comment by GeorgiaHCA** [comment]
Heya @lottycoupat - I know you're really busy so no worry if not - but it would be amazing if you could have a quick look over this for links, spelling and formatting before I give it to @KirstieJane for review.

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 1]
```diff
@@ -0,0 +1,899 @@
...
+**Location**	Local/Municipal/Regional
+**Primary role**	Oral presentation
+Description	A specialist meeting taking place in The Alan Turing Institute consisting of academics and professional practitioners specialising in using data to better understand mental health. 
+
+**Position of attendee** Research Associate 
+**Conference Name**	'Can opening up research help drive innovation?''
+Start Month	Nov
+Start Year	2019
+Location	Local/Municipal/Regional
+URL	https://sustainabilitycommunity.springernature.com /posts/54545-can-opening-up-research-help-drive-in novation
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:127]: innovation

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 2]
```diff
@@ -0,0 +1,899 @@
...
+**Select type of technical product.**	
+Webtool/Application
+**Short title or name for the product**.	
+AutisticaCitizenScience repository
+**Licenced**	
+Yes (Open Source)
+**Open Source license**	
+Other
+**Briefly describe technical product.**	
+This repository holds project managemet tools, resources, and protocols for the Autistica project and provides an open audit of the project's progression and community involvement.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:170]: management

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 3]
```diff
@@ -0,0 +1,899 @@
...
+**Short title or name for the product**.	
+AutSPACEs repository 
+**Licenced**	
+Yes (Open Source)
+Open Source license	MIT license
+**Briefly descrive technical product**.	
+This repository holds the code for the Autistica project and provides an open audit of the project's progression and community involvement.
+**Select the year that this output was realised.**	2020
+**Briefly describe notable impacts** This repository is being used to develop the platform. 
+It includes a requirements list based on thourough community input from a diverse range of autistic participants, and issues derived from those requirements which support the platform development.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:192]: thorough

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 4]
```diff
@@ -0,0 +1,899 @@
...
+Project URL	 
+ 
+## Research Tools and Methods
+
+**Material Type**	Improvements to research infrastructure
+**Material Name**	Participatory co-creation of a qualitative dataset. 
+**Description**	The creation of the Focus Groups dataset was done using a new participatory method. A key was developed to highlight relevant categories individuals fall into as part of the dataset, a decision resulting from discussion sessions with the autistic community. The key was co-designed with Otis Smith, an autistic citizen scientist. In particular, Smith innovated a tag to distinguish between the direct and indirect reporting of experiences. 
+**Provided to Others**	Yes
+**Year First Provided	**2019
+**Impact Description**	Diffrentiating betwen direct and indirect reporting is not a standard practice in qualitative research anaysis, but coud be a useful way of addressing a common complaint by autistic people that their experiences are often spoken about or over by others, and that this prevents them from being directly consulted and listened to.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:221]: differentiating

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 5]
```diff
@@ -0,0 +1,899 @@
...
+The value of making a distinction between direct and indirect reporting has been attested to by autistic people in subsequent discussion sessions. 
+URL	https://github.com/alan-turing-institute/Autistica CitizenScience/tree/master/community-recommendatio ns/focus-groups
+Digital ID	
+Source	Manual
+Publication ID	60995324995342.41724166
+ 
+**Material Type**	Improvements to research infrastructure
+**Material Name**	Collaborative, participatory, open citizen science 
+**Description**	Designing a research methodology which supports autistic people to participate in an empowered way in open research to co-create a citizen science platform. 
+This has required using a broad diversity of tools and protocols for research including tools such as GitHub, google forms, newsletters, individual Zoom sessions, co-wowking sessions, co-authoring, focus group consultations, and workshops.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:231]: co-working

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 6]
```diff
@@ -0,0 +1,899 @@
...
+**In-kind contribution**	
+Yes
+**In-kind contribution currency**	
+GBP British Pound Sterling
+**Give an estimate of the in-kind value.**	
+14,000
+**Contributions Made**	
+We have given a civil servant on the Digital Data and Technology (DDAT) civil service fast stream a six month secondment opportunity. 
+We have provided her with one to one mentoring, opportunities for professional development, direct ownership over tasks, access to new communities and resources, an introduction to new areas of research, and experience and support with a number of technologies and online best practices. 
+**Partner Contributions** The fast streamer seconded to The Alan Turing Institute has provided invaluable support in community management annd project management. She has worked strategically to provide high-level planning as well as detailed task management. Thus, via. this secondment, the civil service is providing 6 months of pro bono support from an exceptional worker which has already resulted in substantial benefits.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:260]: no full stop after via

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 7]
```diff
@@ -0,0 +1,899 @@
...
+**Year Commenced**	2021
+**Year Ended**	Still Active
+URL /	
+**Resultant Outcomes**	The secondment is in the early stages, however outputs include two succesfully run community co-working sessions, a number of high-quality strategic documents for community invovlement, project promotion, and future steps; important management support; and consistently high-quality, dependable support with a number of tasks including repository management and community management.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:264]: involvement

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 8]
```diff
@@ -0,0 +1,899 @@
...
+ 
+**Collaboration Title**	Expert contributions from CRI 
+Partner
+Organisation Name	Interdisciplinary Research Centre
+Contributed Financially	No
+In-kind contribution	Yes
+In-kind contribution currency	EUR Euro
+Give an estimate of the in-kind value.	2000
+Contributions Made	We have shared our research with CRI, and provided them with a full presentation of the project and its aims by the Principle Investigator Kirstie Whitaker. We have also provided developmental internship opportunities for CRI students: doctoral students Katharina Kloppenborg and Sowmya Raja, who have both been offered substantial support and mentorship from the research team.   
+Partner Contributions	CRI has provided us with pro bono support from two highly skilled students at a doctorl level to run user-testing and develop the platform. They have also provided opportunities to disseminate the project's work and values which has been mutually beneficial.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:276]: doctoral

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 9]
```diff
@@ -0,0 +1,899 @@
...
+Partner Contributions	CRI has provided us with pro bono support from two highly skilled students at a doctorl level to run user-testing and develop the platform. They have also provided opportunities to disseminate the project's work and values which has been mutually beneficial. 
+Year Commenced	2020
+Year Ended	Still Active
+URL	
+Resultant Outcomes	Openly published user testing results with diverse autistic participants, as well as documentation for how other researchers could run them. https://github.com/alan-turing-institute/Autistica CitizenScience/tree/master/community-recommendatio ns/user-tests.   Presentation for CRI on "Participatory citizen science to improve autistic people's lives'. DOI: 10.5281/zenodo.3766923.   Ongoing development support for AutSPACEs platform by skilled intern.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:280]: a skilled intern

Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:280]: community-recommendations

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 10]
```diff
@@ -0,0 +1,899 @@
...
+**Contribution Amount**	800
+**In-kind contribution**	Yes
+**In-kind contribution currency**	GBP British Pound Sterling
+**Give an estimate of the in-kind valu**e	333,000
+**Contributions Made**	We have provided Fujitsu with a significant amount of researcher time. 
+We have offered opportunities for development and upskilling for Fujitsu staff in open development, data management, GitHub, neurodiversity and participatory co-creation. 
+We have also given Fujitsu an opportunity to help meet their own charity partnership objective, and supported them to deliver a publiic presentation about their work for Autistica's 2020 Autism Research Festival. 
+**Partner Contributions**
+Fujitsu have provided a team of agile developers to work pro bono on the platform. 
+They have supported autistic people by remunerating them to take part in discussion sessions and user testing, as **well** as hosting these sessions.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:299]: renumerating

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 11]
```diff
@@ -0,0 +1,899 @@
...
+**Department**	Fujitsu Laboratories of Europe
+**Contributed Financially**	Yes
+**Contribution Currency**	GBP British Pound Sterling
+**Contribution Amount**	800
+**In-kind contribution**	Yes
+**In-kind contribution currency**	GBP British Pound Sterling
+**Give an estimate of the in-kind valu**e	333,000
+**Contributions Made**	We have provided Fujitsu with a significant amount of researcher time. 
+We have offered opportunities for development and upskilling for Fujitsu staff in open development, data management, GitHub, neurodiversity and participatory co-creation. 
+We have also given Fujitsu an opportunity to help meet their own charity partnership objective, and supported them to deliver a publiic presentation about their work for Autistica's 2020 Autism Research Festival.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:296]: public

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 12]
```diff
@@ -0,0 +1,899 @@
...
+**Formally Governed**	Yes
+ 
+**Collaboration Title**	OLS for Turing
+**Partner Organisation Name** Open Life Science Limited
+**Contributed financially** No
+**In-kind contribution** Yes
+**In-kind contribution currency** GBP British Pound Sterling
+**Give an estimate of the in-kind value.**	20,000
+**Contributions Made**	Six Turing community members led projects as part of the second cohort of the Open Life Science mentorship programme.    
+Malvika Sharan, community manager for the Turing Way, facilitated and delivered trainings along side the other OLS co-founders.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:320]: alongside

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 13]
```diff
@@ -0,0 +1,899 @@
...
+* Good information, training and support
+* Building on existing relationships
+* Sufficient resources to support effective involvement
+* Enthusiasm and commitment of all involved
+
+**Other factors to successful involvement**
+
+**Challenges of involvement** 
+* Nature of condition can affect ability to be involved
+* Small number of people or appropriate people are difficult  to engage
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:383]: extra space before to engage

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 14]
```diff
@@ -0,0 +1,899 @@
...
+* Lack of resources to support effective involvement
+* Difficulty in finding appropriate ways to communicate and engage,Concerns about how 'representative' views are
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:385]: should be full stops after engage and are

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 15]
```diff
@@ -0,0 +1,899 @@
...
+**Why was involving people challenging?**	
+Difference made by patient involvement	
+Our research is fundamentally participatory, and we have actively engaged and recruited members of the affected community. 
+This is a community of people on the autism spectrum, their relatives, carers, and allie.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:389]: allies

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 16]
```diff
@@ -0,0 +1,899 @@
...
+**Challenges of involvement** 
+* Nature of condition can affect ability to be involved
+* Small number of people or appropriate people are difficult  to engage
+* Lack of resources to support effective involvement
+* Difficulty in finding appropriate ways to communicate and engage,Concerns about how 'representative' views are
+**Why was involving people challenging?**	
+Difference made by patient involvement	
+Our research is fundamentally participatory, and we have actively engaged and recruited members of the affected community. 
+This is a community of people on the autism spectrum, their relatives, carers, and allie. 
+Their work has been core to a number of dimensions: determining the research question; setting the direction of the project; inputting their concerns and desired outputs; designing the methods of the study, co-authoring papers, attending stakeholder meetings, and strategising methods.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:390]: use either semi-colons or commas not both

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 17]
```diff
@@ -0,0 +1,899 @@
...
+* Small number of people or appropriate people are difficult  to engage
+* Lack of resources to support effective involvement
+* Difficulty in finding appropriate ways to communicate and engage,Concerns about how 'representative' views are
+**Why was involving people challenging?**	
+Difference made by patient involvement	
+Our research is fundamentally participatory, and we have actively engaged and recruited members of the affected community. 
+This is a community of people on the autism spectrum, their relatives, carers, and allie. 
+Their work has been core to a number of dimensions: determining the research question; setting the direction of the project; inputting their concerns and desired outputs; designing the methods of the study, co-authoring papers, attending stakeholder meetings, and strategising methods. 
+As such, their involvement has been critical to the research direction, content, and progress.    
+We remain committed to research for and by autistic people, and have been meticulous in ensuring diverse autistic people, including those with complex needs and co-occuding conditions, are involved throughout in a way which is respectful and empowering.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:392]: co-occuring conditions

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 18]
```diff
@@ -0,0 +1,899 @@
...
+**Why was involving people challenging?**	
+Difference made by patient involvement	
+Our research is fundamentally participatory, and we have actively engaged and recruited members of the affected community. 
+This is a community of people on the autism spectrum, their relatives, carers, and allie. 
+Their work has been core to a number of dimensions: determining the research question; setting the direction of the project; inputting their concerns and desired outputs; designing the methods of the study, co-authoring papers, attending stakeholder meetings, and strategising methods. 
+As such, their involvement has been critical to the research direction, content, and progress.    
+We remain committed to research for and by autistic people, and have been meticulous in ensuring diverse autistic people, including those with complex needs and co-occuding conditions, are involved throughout in a way which is respectful and empowering. 
+Our ongoing platform development is also rigorously based on recommendations from the community.     
+Furtheremore, a key research goal has been to empower autistic participants through their involvement.
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:394]: furthermore

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 19]
```diff
@@ -0,0 +1,899 @@
...
+As such, their involvement has been critical to the research direction, content, and progress.    
+We remain committed to research for and by autistic people, and have been meticulous in ensuring diverse autistic people, including those with complex needs and co-occuding conditions, are involved throughout in a way which is respectful and empowering. 
+Our ongoing platform development is also rigorously based on recommendations from the community.     
+Furtheremore, a key research goal has been to empower autistic participants through their involvement. 
+There have been some highly significant outcomes for autistic individuals. Here is a selection:     
+	* Autistic adult with epilepsy: after setbacks following illness and coronavirus disruptions, being inspired to enter into autism research, applying for a master's course, and delivering own research at a postgraduate-level conference.  
+	* Autistic adult with dyspraxia: becoming a neurodiversity advocate at thier workplace and being inspired to apply for a secondment to The Alan Turinig Institute .
```
Comment (Angelsaur) [inline project-communications/academic-updates/researchfish-updates-2021.md:397]: less space before fullstop

File: project-communications/academic-updates/researchfish-updates-2021.md (hunk 2 above)
Comment (GeorgiaHCA) [reply project-communications/academic-updates/researchfish-updates-2021.md]: Done!

File: project-communications/academic-updates/researchfish-updates-2021.md (hunk 10 above)
Comment (GeorgiaHCA) [reply project-communications/academic-updates/researchfish-updates-2021.md:299]: it is remunerating ;) althuogh it seems like it should be renumerating becuase of numerical - such a confusing word!

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 20]
```diff
@@ -0,0 +1,899 @@
...
+Yes (Open Source)
+**Open Source license**	
+Other
+**Briefly describe technical product.**	
+This repository holds project management tools, resources, and protocols for the Autistica project and provides an open audit of the project's progression and community involvement. 
+**Select the year that this output was realised.**	2019
+**Briefly describe notable impacts.**	The repository has been used for project management and to collaborate with autistic citizen scientists, offering important upskilling opportunities. 
+It also allows the ongoing work to be publicly available and easily shared with community members. 
+This supports the open and participatory values of the project. 
+**URL.**	https://github.com/alan-turing-institute/Autistica CitizenScience
```
Comment (lottycoupat) [inline project-communications/academic-updates/researchfish-updates-2021.md:175]: there's a space here in the URL

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 21]
```diff
@@ -0,0 +1,899 @@
...
+**In-kind contribution currency** GBP British Pound Sterling
+**Give an estimate of the in-kind value.**	20,000
+**Contributions Made**	Six Turing community members led projects as part of the second cohort of the Open Life Science mentorship programme.    
+Malvika Sharan, community manager for the Turing Way, facilitated and delivered trainings along side the other OLS co-founders. 
+**Partner Contributions**	Open Life Science (https://openlifesci.org/) program mentors and trains individuals and stakeholders working in research organisations who want to promote Open Science practice in their communities. 
+Participants of this 16-week mentoring & training program learn essential knowledge required to create, lead, and sustain an Open Science project, connect with members across different communities, backgrounds, and identities by sharing their experiences and expertise. 
+The complete schedule and participation guidelines of the program are available at https://openlifesci.org/syllabus.   
+**Year Commenced**	2020
+**Year Ended**	Still Active
+URL	https://github.com/alan-turing-institute/the-turin g-way/tree/master/open-life-science-mentoring
```
Comment (lottycoupat) [inline project-communications/academic-updates/researchfish-updates-2021.md:326]: is there a space also here?

File: project-communications/academic-updates/researchfish-updates-2021.md [hunk 22]
```diff
@@ -0,0 +1,899 @@
...
+The chapter is being co-written with autistic researcher and strategist who joined the projcet as a citizen scientist.      
+* Findings and learning from the project have been shared with groups throughout The Alan Turing Institute. 
+A participatory science co-working group was set up with researchers from Google DeepMind and learning from the project was shared with members. 
+Findings and learning from the project have been shared with groups throughout The Alan Turing Institute. 
+A participatory science co-working group was set up with researchers from Google DeepMind and learning from the project was shared with members. 
+The project's research associate became a co-lead of the Social Data Sciences Special Interest Group and has presented this project with group members, as well as a facilitating collaborator from The Open Humans Foundation to share knowledge with The Turing.
+
+## Participatory Science    
+We remain committeed to research for and by autistic people, and have been highly meticulous in ensuring diverse autistic people, including those with complex needs or co-occuding conditions, are involved throughout in a way which is respectful and empowering.
```
Comment (lottycoupat) [inline project-communications/academic-updates/researchfish-updates-2021.md:511]: typo - committed

**Review by lottycoupat — APPROVED** [review]

**Comment by GeorgiaHCA** [comment]
Thanks @lottycoupat !

**Review by KirstieJane — COMMENTED** [review]
Thank you for this @GeorgiaHCA - have you looked at the rendered version of the markdown document? A lot of the sections run together in big paragraphs when I would have expected them to have bullet points - there are also a lot of other formatting errors. Let me know if you need / want me to have a pass through the document (if you're short on time) or if you'd like to have a pass through first.

**Comment by GeorgiaHCA** [comment]
Hey - thanks for taking a look @KirstieJane ! I do think it look super awkward in markdown - it's following the formatting of the form in a way which really isn't all that readable - sorry <3 - how about taking a look at the form on Researchfish itself while I work more on the formatting in mark down? That way it can be submitted but also can look much cleaner for the repo? Don't worry, I def do have time for another pass at it :)

**Pull request merged**